pypesto/sample/parallel_tempering.py

```diff
@@ -43,12 +46,16 @@ def __init__(
         # set betas
         if (betas is None) == (n_chains is None):
             raise ValueError("Set either betas or n_chains.")
-        if betas is None:
+        if betas is None and self.options["beta_init"] == "exponential_decay":
             betas = near_exponential_decay_betas(
                 n_chains=n_chains,
                 exponent=self.options["exponent"],
                 max_temp=self.options["max_temp"],
             )
+        elif betas is None and self.options["beta_init"] == "beta_decay":
```

Comment:
Some constant in `C.py` for `"beta_decay"` and `"exponential_decay"` would be nice